test/forms/WebsiteFormProviderSpec.scala

```diff
@@ -37,9 +41,15 @@ class WebsiteFormProviderSpec extends StringFieldBehaviours {
     behave like fieldThatBindsValidData(
       form,
       fieldName,
-      stringsWithMaxLength(maxLength)
+      validData
     )
 
+    "must not bind invalid website data" in {
+      val invalidWebsite = "invalid"
```

Comment:
again could we add a couple failing scenarios to check websites with missing .something ? And www. (If that is mandatory)